### DISCUSSION TITLE
It doesn't work ... why

### DISCUSSION THREAD

**Discussion by Bersaker** [valinet/ExplorerPatcher#182] [Q&A]
hello, i reinstalled windows 11. I use the version Windows Feature Experience Pack 1000.22000.282.0
I downloaded the file dxgi.dll release 22000.282.31.3. copy this into the folder c: / windows

then I use win x ... run the task manager, search for explorer.exe and restart this file.
afterwards nothing happens anymore. there is no pop up window on the right side ...
what am I doing wrong?

**Comment by yonderbread** [comment]
I have the exact same problem, anyone know why this is happening?
Edit: Wait, nevermind! Literally just as I finished sending this message my taskbar updated lmao
